### PR TITLE
push1_1 python

### PR DESCRIPTION
**Pull request by D-E-F-E-A-T**
python up

### CODE REVIEW

**Comment by D-E-F-E-A-T** [comment]
backend (django and flask) and python

**Comment by D-E-F-E-A-T** [comment]
django and flask in backend,
security
python

**Comment by D-E-F-E-A-T** [comment]
updates:
django, flask in backend
security
python
flutter kotlin (very less) in mob dev
ml
html and css (bulk) in frontend

**Comment by D-E-F-E-A-T** [comment]
Links from readme.md directing to folders now... will be making a static page for it right?

**Comment by D-E-F-E-A-T** [comment]
i uploaded resources that I had... if more needed ping i will find more

**Comment by D-E-F-E-A-T** [comment]
update: reverse eng

File: Backend/Backend.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+## Backend Resources
+
+#### Django
+
+* [Django Girls Tutorial](https://tutorial.djangogirls.org/en/) (1.11)
+* [Django Official Documentation](https://media.readthedocs.org/pdf/django/1.5.x/django.pdf) (PDF) (1.5)
+* [Django Official Documentation](https://media.readthedocs.org/pdf/django/1.7.x/django.pdf) (PDF) (1.7)
+* [Django Official Documentation](https://media.readthedocs.org/pdf/django/1.9.x/django.pdf) (PDF) (1.9)
+* [Django Official Documentation](https://media.readthedocs.org/pdf/django/1.10.x/django.pdf) (PDF) (1.10)
+* [Django Official Documentation](https://buildmedia.readthedocs.org/media/pdf/django/2.2.x/django.pdf) (PDF) (2.2)
```
Comment (Geek-ubaid) [inline Backend/Backend.md]: Too many Django versions documentation. There is not much difference i suppose

File: Python/Python.md [hunk 2]
```diff
@@ -1 +1,90 @@
...
+* [The Definitive Guide to Jython, Python for the Java Platform](http://www.jython.org/jythonbook/en/1.0) - Josh Juneau, Jim Baker, Victor Ng, Leo Soto, Frank Wierzbicki (2.5)
+* [The Little Book of Python Anti-Patterns](http://docs.quantifiedcode.com/python-anti-patterns/) ([Source](https://github.com/quantifiedcode/python-anti-patterns))
+* [The Programming Historian](http://niche-canada.org/research/niche-digital-infrastructure-project/the-programming-historian/) - William J. Turkel, Adam Crymble and Alan MacEachern
+* [The Python Ecosystem: An Introduction](http://mirnazim.org/writings/python-ecosystem-introduction/)
+* [The Python GTK+ 3 Tutorial](http://python-gtk-3-tutorial.readthedocs.org/en/latest/)
+* [The Standard Python Library](http://effbot.org/librarybook/) - Fredrik Lundh
+* [Think Complexity](http://greenteapress.com/complexity/) - Allen B. Downey (2nd Edition) (PDF, HTML)
+* [Tiny Python 3.6 Notebook](https://github.com/mattharrison/Tiny-Python-3.6-Notebook) - Matt Harrison (3.6)
+* [Web2py: Complete Reference Manual, 6th Edition (pre-release)](http://web2py.com/book) (2.5 - 2.x)
```
Comment (Geek-ubaid) [inline Python/Python.md]: Too many resources added. People might get confused. Include only those resources that are really very good

**Comment by Geek-ubaid** [comment]
Too many resources of each domain added. Try to include only those resources that are really good. You can include courses that you have done. People will get confused and overwhelmed seeing too much to learn.

**Comment by D-E-F-E-A-T** [comment]
@Geek-ubaid working on it

File: Python/Python.md (hunk 2 above)
Comment (D-E-F-E-A-T) [reply Python/Python.md]: OK

**Comment by D-E-F-E-A-T** [comment]
@Geek-ubaid I have removed many from python and frontend. For security it's table formation(see  actual from forked repo) so it's not that big as it's about tools etc. For ML I would request you to make changes

**Comment by D-E-F-E-A-T** [comment]
> @Geek-ubaid I have removed many from python and frontend. For security it's table formation(see actual from forked repo) so it's not that big as it's about tools etc. For ML I would request you to make changes

See if more reduction is needed then I would remove to only few

**Comment by Geek-ubaid** [comment]
Okkay will review the changes

**Pull request merged**